admin_manual/maintenance/upgrade.rst

```diff
@@ -2,93 +2,47 @@
 How to Upgrade Your ownCloud Server
 ===================================
 
-There are three ways to upgrade your ownCloud server:
+We recommend that you keep your ownCloud server upgraded regularly installing all point releases and major releases, without skipping any of them, as doing so increases the risk of errors. 
+
+- Major releases are 8.0, 8.1, 8.2, 9.0, and 10.0. 
```

Comment:
This could have 9.1 and 9.2 added to the list of "major" releases. Without it, it made me think for a minute if there is something different about the release numbering from 9.0 onward. But I don't think there is.